resources/styles/book-content/ap-connection.less

```diff
@@ -0,0 +1,9 @@
+.ap-connection {
+  > h1:first-child {
+    // The "Connection for AP Courses" message appears on every page and is redundant.
+    // although we don't want it to display, it still should take up space in order to provide a top margin
+    height: 0;
+    margin: 30px;
+    visibility: hidden;
```

Comment:
Yes, I know we discussed it quite a bit, but thought we'd ultimately decided to just hide the heading.  The note don't mention it at all :frowning: